cool.c

```diff
@@ -0,0 +1,7 @@
+#include<stdio.h>
```

Comment:
good code try putting comments from next time, so that everyone understands your code 👍 💯 🥇